baybe/kernels/composite.py

```diff
@@ -0,0 +1,74 @@
+"""Composite kernels."""
```

Comment:
Add short description what we mean by "composite".

Reply:
done